config.json

```diff
@@ -16,6 +16,16 @@
     {
       "core": true,
```

Comment:
The exercise is very simple, so it's ok to place it just after `hello-world`. But, I wouldn't configure it as `core`. `core` exercises where already chosen some time ago.

I would configure it as `core: false`. In that case, the key `unlocked_by` must be provided. It could be unlocked by `hello-world`. Both exercises are very similar in complexity I think.